rmw_zenoh_cpp/src/detail/graph_cache.cpp

```diff
@@ -612,9 +611,10 @@ void GraphCache::parse_del(
 
   if (entity->type() == EntityType::Node) {
     // Node
-    // The liveliness tokens to remove pub/subs should be received before the one to remove a node
-    // given the reliability QoS for liveliness subs. However, if we find any pubs/subs present in
-    // the node below, we should update the count in graph_topics_.
+    // In case the remote Node closed abruptly or was disconnected, Zenoh could deliver the
+    // liveliness tokens unregistration events in any order.
+    // If the event for Node unregistration comes before the unregistration of its
+    // pubs/subs/services, we should update the count in graph_topics_ and graph_services_.
```

Comment:
Suggest to rephrase as follows
```suggestion
    // When destroying a node, Zenoh does not guarantee that liveliness tokens to remove pub/subs
    // arrive before the one to remove the node from the graph despite un-registering those entities earlier.
    //  In such scenarios, if we find any pub/subs present in the node, we reduce their counts in graph_topics_.
```